api/lib/source/polkadotV0-source.js

```diff
@@ -693,9 +693,14 @@ class polkadotAPI {
       )
     }
     if (type === `treasury`) {
-      description = `This is a Treasury Proposal whose description and title have not yet been edited on-chain. Only the proposer address (${
-        proposal.proposer || proposer
-      }) is able to change it.`
+      const { meta } = proposal.council[0]
```

Comment:
what do we need the meta for? the description should stay the same as the treasury proposal

Reply:
Jordan and I just talked that there should be a description coming from the chain (for this we need `meta`) and the other default one that is custom from us in a separate field (`note`).

But this change will be done in another PR. Yes, we need `meta`

Reply:
we use the description of the treasury proposal, not from the council. so please revert the change

Reply:
it is the same if you look at the object (OK, we need to wait until there are new Treasury proposals, but you can see here it is the same Object):

```
{
"council": [],
"id": 9,
"proposal": {
"proposer": "16XXocb76BpprdspsR7j5iWFHf4aTMnsVHxgGBXfixs1FFLb",
"value": 70000000000000,
"beneficiary": "16XXocb76BpprdspsR7j5iWFHf4aTMnsVHxgGBXfixs1FFLb",
"bond": 3500000000000
}
}
```

Reply:
Like you said, they are actually the same treasury and council, and council it is just a way of voting, but the description is the same since they are both the same proposal

Reply:
the description is actually not the same

Reply:
Are you sure about that? It should be. I will check myself

Reply:
It is exactly the same description. You can check yourself by copy-pasting this on the Javascript section in polkadot.js

```
const councilProposals = await api.derive.council.proposals()
const treasuryProposals = await api.derive.treasury.proposals()

console.log(JSON.stringify(councilProposals))
console.log(JSON.stringify(treasuryProposals))

const councilDescriptionMeta = await api.registry.findMetaCall(councilProposals[0].proposal.callIndex)
const councilDescription = councilDescriptionMeta.meta.documentation.toString()

const treasuryDescriptionMeta = await api.registry.findMetaCall(treasuryProposals.proposals[0].council[0].proposal.callIndex)
const treasuryDescription = treasuryDescriptionMeta.meta.documentation.toString()

console.log('\n COUNCIL DESCRIPTION', councilDescription)
console.log('\n TREASURY DESCRIPTION', treasuryDescription)
```